docs/elements.md

```diff
@@ -1255,6 +1255,45 @@ Note that some Python types may not be serialized correctly in the code provided
 - [`pl-variable-output` for displaying a matrix or element in code form.](#pl-variable-output-element)
 - [`pl-python-variable` for displaying a formatted output of Python variables.](#pl-python-variable-element)
 
+### `pl-details` element
+
+Displays question content within a collapsible accordion-style component.
+
+![pl-details element closed](elements/pl-details-closed.png)
+![pl-details element opened](elements/pl-details-open.png)
+
+#### Sample element
+
+```html title="question.html"
+<pl-question-panel>
+  <pl-details title="Hint">
```

Comment:
I don't think this is a great example, since something like this should be done with the existing hint element.

Reply:
Changed the language to "Additional information". 3468146f0a5d30540d37d68c2985e87c5c7eaef7